Implemented Login Screen with Reusable Components

Tole15 opened the pull request:
This commit introduces the Login Screen, which has been developed using a reusable component approach. By leveraging reusable components, we have improved code efficiency, enhanced design consistency, and ensured easier maintenance of the application.

The Login Screen incorporates various reusable components that contribute to its overall design and functionality. These components have been carefully crafted to serve specific purposes, resulting in a cohesive and user-friendly login experience.

By employing a reusable component strategy, we have achieved a more streamlined development process, allowing for faster iteration and updates in the future. This approach also promotes code reusability and ensures a consistent design language across different screens of the application.
![Screenshot_1688110376](https://github.com/BrightCoders-Institute/reto-reservacion-de-vuelos-equipo-alfa-buena-maravilla-onda/assets/129608820/a1d644d0-4085-4ec2-9d5c-66b6e4b8ffd0)

Tole15 committed:
Implemented Login Screen with Reusable Components

This commit introduces the Login Screen, which has been developed using a reusable component approach. By leveraging reusable components, we have improved code efficiency, enhanced design consistency, and ensured easier maintenance of the application.

The Login Screen incorporates various reusable components that contribute to its overall design and functionality. These components have been carefully crafted to serve specific purposes, resulting in a cohesive and user-friendly login experience.

By employing a reusable component strategy, we have achieved a more streamlined development process, allowing for faster iteration and updates in the future. This approach also promotes code reusability and ensures a consistent design language across different screens of the application.

Co-authored-by: BrandonVG <brandon.vadillox3@gmail.com>
Co-authored-by: Axw3ll <axelomar.a.v@gmail.com>

DerekAyala reviewed (approved):
LGTM!

montsegv-2 reviewed (approved):
LGTM, good job!

Eduardoespl commented on src/components/LogInForm.jsx:
```diff
@@ -0,0 +1,49 @@
+    setEmail,
+    password,
+    setPassword,
+    termsChecked,
+    setTermsChecked,
+    subscribeChecked,
+    setSubscribeChecked,
```
U need this lines in this file?

Eduardoespl reviewed (approved):
LGTM!

Ewar22 reviewed (approved):
Nice

Tole15 committed:
Enhancements to Login Screen

Added more details and instructions to the Login Screen, implemented an animated company logo, and enabled default button functionality for improved user experience. These updates enhance the screen's appearance, interaction, and branding.

Co-authored-by: BrandonVG <brandon.vadillox3@gmail.com>
Co-authored-by: Axw3ll <axelomar.a.v@gmail.com>

GustavoHiguera reviewed (approved):
LGTM
